zerver/data_import/rocketchat.py

```diff
@@ -802,7 +802,8 @@ def map_upload_id_to_upload_data(
         upload_id_to_upload_data_map[upload["_id"]] = {**upload, "chunk": []}
 
     for chunk in upload_data["chunk"]:
-        upload_id_to_upload_data_map[chunk["files_id"]]["chunk"].append(chunk["data"])
+        if chunk["files_id"] in upload_id_to_upload_data_map:
+            upload_id_to_upload_data_map[chunk["files_id"]]["chunk"].append(chunk["data"])
```

Comment:
I'd probably go with inverting the if statement and logging the anomaly here, rather than silently doing nothing.

Reply:
Resolved in the meantime by 28a29e64a0a75c3475ca9cb7cae0815620bd095c